docs/sdk/tutorials/importing_video_assets.md

```diff
@@ -222,6 +223,39 @@ assets = kili.append_many_to_dataset(
 )
 ```
 
+## Updating video metadata
+
+Except the processingParameters, json metadatas aren't persistent, this means that if you want to update one metadata you will have to pass all the others metadatas to keep them, otherwise it will be erased.
```

Comment:
A suggestion to make things less ambiguous:
When importing a video, if you want to add or delete metadata, you must provide the entire list of metadata. This new list will replace the previous one, meaning any metadata not included in the new list will be erased. However, this does not apply to the `processingParameters` metadata, which will be retained even if they are not present in the new list.

Reply:
That's better thanks, i've update the doc